bin/grade_students.sh

```diff
@@ -857,23 +855,31 @@ while true; do
 
 
 	# -------------------------------------------------------------
-        # create/append to the results history
-        sec_deadline=`date -d "${global_assignment_deadline}" +%s`
-        sec_submission=`date -d "${global_submission_time}" +%s`
-        seconds_late=$((sec_submission-sec_deadline))
-        /usr/local/submitty/bin/grade_students__results_history.py  \
-            "$global_results_history_file_location" \
-            "$global_assignment_deadline" \
-            "$global_submission_time" \
-            "$seconds_late" \
-             "`date -d @$FILE_TIMESTAMP`" \
-            "$IS_BATCH_JOB" \
-            "`date -d @$STARTTIME`" \
-            "$WAITTIME" \
-            "`date -d @$ENDTIME`" \
-            "$ELAPSED" \
-	    "$global_grade_result"
-
+    # create/append to the results history
+    sec_deadline=`date -d "${global_assignment_deadline}" +%s`
+    sec_submission=`date -d "${global_submission_time}" +%s`
+    seconds_late=$((sec_submission-sec_deadline))
+    ${SUBMITTY_INSTALL_DIR}/bin/grade_students__results_history.py  \
+        "$global_results_history_file_location" \
+        "$global_assignment_deadline" \
+        "$global_submission_time" \
+        "$seconds_late" \
+         "`date -d @$FILE_TIMESTAMP`" \
+        "$IS_BATCH_JOB" \
+        "`date -d @$STARTTIME`" \
+        "$WAITTIME" \
+        "`date -d @$ENDTIME`" \
+        "$ELAPSED" \
+        "$global_grade_result"
+
+    #---------------------------------------------------------------------
+    # WRITE OUT VERISON DETAILS
+    ${SUBMITTY_INSTALL_DIR}/bin/insert_database_version_data.py \
+        "${semester}" \
+        "${course}" \
+        "${assignment}" \
+        "${user}" \
+        ${version}
```

Comment:
I'm not sure I like that these variables are being pulled out of grade_item function, but this has to run after the above python script to get the submission date. I think if this ever got rewritten, we'd improve the way things are handled with these variables, but probably not a big deal for now?